### DISCUSSION TITLE
Issue with copilot not working with remote host

### DISCUSSION THREAD

**Discussion by eugfomitcheva** [community/community#123330] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

I am unable to get copilot inline suggestions working when using a remote host, even though it's properly installed in both local and remote. I've spent hours uninstalling/reinstalling, clearing the vscode dir etc. to no avail. Can I please get some help?

Version is latest

**Comment by devm33** [comment, marked as answer]
This was likely the same issue as here https://github.com/orgs/community/discussions/123301 Should be resolved on 1.190.0

**Comment by brandonyee-cs** [comment]
If you're using VSCODE, just access copilot via the activity bar.
